.github/workflows/release.yml

```diff
@@ -1,38 +0,0 @@
-name: "release"
-on:
-  push:
-    tags:
-      - 'v*' # Push events to matching v*, i.e. v1.0, v2.1.3
-jobs:
-  announcement:
-    continue-on-error: true
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - run: npm ci
-      - name: Package
-        run: npm run package
-      - name: "Check that packaging didn't create any changes"
-        run: '[ -z "$(git status --porcelain)" ];'
-      - name: Set release message
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
-      - name: Check whether we should send release annoucement
-        id: enable_release_annoucement
-        env:
-          ZULIP_API_KEY: ${{ secrets.ZULIP_API_KEY }}
-        run: |
-          echo "::set-output name=value::${{ env.ZULIP_API_KEY != '' }}"
-      - name: Send a stream message
-        uses: ./
-        with:
-          username: 'ggrossetie@yuzutech.fr'
-          api-key: ${{ secrets.ZULIP_API_KEY }}
-          organization-url: 'https://kroki.zulipchat.com'
-          to: 'test'
-          type: 'stream'
-          topic: 'releases'
-          content: |
-            zulip-send-message-action ${{ env.RELEASE_VERSION }} is out!
-
-            https://github.com/yuzutech/zulip-send-message-action/releases/tag/${{ env.RELEASE_VERSION }}
-        if: ${{ steps.enable_release_annoucement.outputs.value == 'true' }}
```

Comment:
It was an experiment to publish a release announcement when a new release is published.

I guess we could publish an announcement in ["announce" stream](https://chat.zulip.org/#narrow/stream/1-announce) but in the meantime I've decided to remove it since the API key and organization URL are outdated.